hacks_mbof/settings.py

```diff
@@ -164,7 +164,7 @@
             'level': 'DEBUG',
         },
         'django.db.backends': {
-            # 'level': 'DEBUG', # Uncomment to log all SQL statements
+            'level': 'DEBUG', # Uncomment to log all SQL statements
```

Comment:
`settings.py` is definitely a properties file.  It can read environment variables.  So, for example, where this logging level is set, we could use:

```python
'level': 'DEBUG' if (os.getenv('DJANGO_SQL_DEBUG') == 'DEBUG') else None
```